### ISSUE TITLE
why restrict multisig operations to REV transfers?

### ISSUE THREAD

**Issue by dckc** [fabcotech/rchain-multisig#3]
why restrict multisig operations to REV transfers?

Surely there are many other interesting use cases, and the general case is that of collectively deciding to send some message to some shared power.

This is the design in WorkingGroup.rho
https://github.com/rchain-community/rgov/blob/master/WorkingGroup.rho

Or if you just want to do REV transfers, why not the existing MultiSigRevVault.rho ?
https://github.com/rchain/rchain/blob/dev/casper/src/main/resources/MultiSigRevVault.rho

**Comment by fabcotech** [comment]
Currently two operations can be performed `TRANSFER_REV` and `ACCEPT` this is in the readme at the end.

Yes we very much plan to support other operations such as `APPLY` which will allow multisigs to participate in other multisigs (resucrusion described in the youtube video).

And then the goal is for a multisig to also be able to manage tokens, which means allowing the RChain token operations.

Sending a message is also a good idea, do you mean sending a message to an arbitrary rholang channel, or just having a log of issues/messages that have been stated/voted at 66% by the multisig contract.

**Comment by dckc** [comment]
> do you mean sending a message to an arbitrary rholang channel

Yes, since that subsumes all other possibilities. The others can be layered on top  The REV channel in question can be one that makes REV transfers.

(Changing membership needs to be in the same layer. But note that WorkingGroup.rho lets the proposer supply any function to modify the membership. The function can empty the membership to shut down the group, for example.)
